src/ubx.cpp

```diff
@@ -725,6 +725,8 @@ int GPSDriverUBX::configureDevice(const GPSConfig &config, const int32_t uart2_b
 		// heading output period 1 second
 		cfgValset<uint8_t>(UBX_CFG_KEY_MSGOUT_UBX_NAV_RELPOSNED_UART1, 1, cfg_valset_msg_size);
 		cfgValset<uint8_t>(UBX_CFG_KEY_MSGOUT_UBX_NAV_RELPOSNED_USB, 1, cfg_valset_msg_size);
+		cfgValset<uint8_t>(UBX_CFG_KEY_MSGOUT_UBX_NAV_HPPOSLLH_UART1, 1, cfg_valset_msg_size);
+		cfgValset<uint8_t>(UBX_CFG_KEY_MSGOUT_UBX_NAV_HPPOSLLH_USB, 1, cfg_valset_msg_size);
```

Comment:
Instead of all of these, can you instead add `cfgValsetPort(UBX_CFG_KEY_MSGOUT_UBX_NAV_HPPOSLLH_I2C, 1, cfg_valset_msg_size);` after https://github.com/PX4/PX4-GPSDrivers/blob/83e56c680a800847dc2b45098e4c220ae09f7497/src/ubx.cpp#L666

Reply:
Added definition of UBX_CFG_KEY_MSGOUT_UBX_NAV_HPPOSLLH_I2C, and the line (667) that allows that message on I2C.
We don't want to disallow HPPOSLLH on UARTs and USB, do we?

Reply:
@bkueng  - thanks for explanation. I cleaned up **all** RELPOSNED and HPPOSLLH port setup, left only I2C as you suggested. Tested on my dual F9P RTK platform - everything works fine, these messages are coming via USB as before.